backend/lib-shared-site-api/src/layer/slack_layer.rs

```diff
@@ -200,9 +204,16 @@ where
         let blocks_result = Self::build_blocks(message, event, metadata);
         match blocks_result {
             Ok(blocks) => {
+                if self.exec_env == ExecEnv::Dev || self.exec_env == ExecEnv::Ci {
```

Comment:
@ClaireMao1996 I disabled slack logging in dev/ci, and printed directly to console. I tried to do it in `worker.rs`, but ran in to some strange problems with the channel closing.